src/centrality.rs

```diff
@@ -39,29 +31,16 @@ use petgraph::visit::{
 // P -- predecessors
 // S -- verts_sorted_by_distance,
 //      vertices in order of non-decreasing distance from s
-// Q -- Q
+// Q -- queue
```

Comment:
This list is meant to give correspondences between the identifier (variable name) used in the code and the mathematical symbol used in the expressions in the journal article. The symbol `Q` is the same in both instances.
There is a slight deviation in that I spell out "sigma" and "delta" rather that using the symbols, as the paper does. I suppose I should actually use a utf-8 character for sigma and delta.

Reply:
Yeah, I renamed the variable `Q` -> `queue` to avoid having `#[allow(non_snake_case)]` and this is reason i updated this list too. If you feel the code is more difficult to read, i can revert the changes.

Reply:
No, I think `queue` is better than `Q`. I can't think of a better name.